src/laser/MultiLaser.cpp

```diff
@@ -533,6 +533,34 @@ MultiLaser::ShiftLaserSlices ()
     }
 }
 
+void
+MultiLaser::GetLaserAabs (const int current_N_level, Fields& fields,
```

Comment:
Would a name like `UpdateLaserAabs` be more appropriate?

Reply:
Yes